### ISSUE TITLE
Why create a new project, instead of just sending a pull request to the original?

### ISSUE THREAD

**Issue by thosakwe** [anuragvohraec/dartJSEngine#1]
https://github.com/samurai-dart/samurai is open for pull requests, and having two identical packages creates fragmentation in the Dart ecosystem.

I was actually going to make a similar comment on your `parsejs` fork the other day, but issues were not open on your fork.

Obviously, a pull request is not required, and it's up to you, but IMO it would be much better in the long run.

**Comment by anuragvohraec** [comment]
When I needed this project,the original project was almost not maintained.
I needed few modifications, in it to work.
So I pulled it and made few of my changes I requried.

Later on, it too become unsuited for my requirement so I wrote this: [JSEngine](https://github.com/anuragvohraec/jsengine).

Now meanwhile few people pulled out my changes and made projects on it, so have to keep this branch live for their project to work.
